core/src/main/scala/cats/syntax/functor.scala

```diff
@@ -62,3 +65,20 @@ final class FunctorTuple2Ops[F[_], A, B](private val fab: F[(A, B)]) extends Any
    */
   def unzip(implicit F: Functor[F]): (F[A], F[B]) = F.unzip(fab)
 }
+
+final class IfFOps[F[_]](private val fa: F[Boolean]) extends AnyVal {
+
+  /**
+   * Lifts `if` to Functor
+   *
+   * Example:
+   * {{{
+   * scala> import cats.Functor
+   * scala> import cats.implicits.catsStdInstancesForList
+   *
+   * scala> Functor[List].ifF(List(true, false, false))(1, 0)
```

Comment:
It is a doc for `Functor`, not for the syntax. The syntax usage should be documented something like:
```suggestion
   * scala> import cats.syntax.all._
   *
   * scala> List(true, false, false).ifF(1, 0)
```